### PR TITLE
[IIIF-463] Add human-friendly responses

### PR DESCRIPTION
**Pull request by DRickard**

### CODE REVIEW

File: src/test/java/edu/ucla/library/bucketeer/handlers/DeleteJobHandlerTest.java [hunk 1]
```diff
@@ -58,7 +58,8 @@ public final void testDeleteJobHandler(final TestContext aContext) {
                                 final HttpResponse<Buffer> response = deletion.result();
                                 final int statusCode = response.statusCode();
 
-                                aContext.assertEquals(HTTP.NO_CONTENT, statusCode, LOGGER.getMessage(
+                                //aContext.assertEquals(HTTP.NO_CONTENT, statusCode, LOGGER.getMessage(
```
Comment (ksclarke) [inline src/test/java/edu/ucla/library/bucketeer/handlers/DeleteJobHandlerTest.java:61]: Did you mean to leave this commented out text in?

Comment (DRickard) [reply src/test/java/edu/ucla/library/bucketeer/handlers/DeleteJobHandlerTest.java:61]: I was experimenting, will remove the commented line before making this an official PR

**Review by ksclarke — APPROVED** [review]
:+1:

**Comment by DRickard** [comment]
Merged in updates from "master," so need refreshed reviews.

**Review by ksclarke — APPROVED** [review]

**Review by markmatney — APPROVED** [review]

**Pull request merged**